script-runner/TestCases.hs

```diff
@@ -171,11 +171,13 @@ test4 targetblocksize expectedResult = do
         endScript $ ExitFailure 3
   on (3,10) $ \Dict _diffusion -> do
     bvd <- gsAdoptedBVData
-    case (bvdMaxBlockSize bvd == targetblocksize) of
-      True -> endScript ExitSuccess
-      _ -> do
-        liftIO $ hPutStrLn stderr "max block size not what was expected"
-        endScript $ ExitFailure 4
+    if bvdMaxBlockSize bvd == targetblocksize
+       then do
+         liftIO $ hPutStrLn stderr "test passed"
```

Comment:
Do we have to write this to stderr for some reason? This might be a message we'd still like to see even when suppressing stderr output, for example (`2> /dev/null`).

Reply:
👍 see other comment [here](https://github.com/input-output-hk/cardano-sl/pull/4061/files#r253485184)